app/controllers/application_controller.rb

```diff
@@ -23,7 +23,7 @@ def after_sign_in_path_for(resource_or_scope)
   end
 
   def guest_uid_authentication_key(key)
-    key &&= nil unless key.to_s =~ /^guest/
+    key &&= nil unless /^guest/.match?(key.to_s)
```

Comment:
I wonder why this is preferred

Reply:
answer: performance, apparently. https://www.rubydoc.info/gems/rubocop/0.47.1/RuboCop/Cop/Performance/RegexpMatch